app/views/layouts/mail/school.html.erb

```diff
@@ -2,48 +2,18 @@
 
 <html lang="en">
   <head>
-    <%= stylesheet_link_tag 'mailer' %>
     <%= stylesheet_link_tag 'https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600' %>
```

Comment:
Do we need this font?

Reply:
I feel the mail looks better with font 🤔 

**With font**
![image](https://user-images.githubusercontent.com/2093818/150475928-80d2ce3a-ad86-4770-b7ff-f92eaee95966.png)
**Without Font**
![image](https://user-images.githubusercontent.com/2093818/150475975-ab62db5f-a78b-467d-88c4-3c889f3d319c.png)

Reply:
This is updated by Vinu [here](https://github.com/pupilfirst/pupilfirst/pull/880#issuecomment-1020567697), closing the conversation.